### ISSUE TITLE
Cleaner crash when Memory error?

### ISSUE THREAD

**Issue by mboudet** [gogepp/genoboo#32]
https://gitlab.com/beaur1s/beauris/-/issues/61

**Comment by mboudet** [comment]
Not sure we can 'cleanly exit', the whole application crashes.

Maybe set a timeout client side so that it would end without response?
(Also, maybe be should make multiple bulk instead of eating all the memory)

**Comment by abretaud** [comment]
yep, probably the galaxy tool should not be too confident on gnb error handling, and maybe monitor gnb.log, or check if gnb daemon is responding somehow
